dpc-api/src/main/java/gov/cms/dpc/api/resources/v1/GroupResource.java

```diff
@@ -351,18 +355,24 @@ private OffsetDateTime handleSinceQueryParam(String since) {
      *
      * @param outputFormat param to check
      */
-    private static void checkExportRequest(String outputFormat, String headerPrefer) {
-        // _outputFormat only supports FHIR_NDJSON
+    private static void checkExportRequest(String outputFormat, String, Prefer String headerAccept) {
+
+      // _outputFormat only supports FHIR_NDJSON
         if (StringUtils.isNotEmpty(outputFormat) && !FHIR_NDJSON.equals(outputFormat)) {
             throw new BadRequestException("'_outputFormat' query parameter must be 'application/fhir+ndjson'");
         }
+        if (headerAccept==null || StringUtils.isEmpty(headerAccept)){
+            throw new BadRequestException("'Accept' header must be 'application/fhir+json'");
+        }
+        if (StringUtils.isNotEmpty(headerAccept) && !FHIR_JSON.equals(headerAccept)) {
+            throw new BadRequestException("'Accept' header must be 'application/fhir+json'");
+        }
```

Comment:
```suggestion
        if (StringUtils.isEmpty(headerAccept) || !FHIR_JSON.equals(headerAccept)){
            throw new BadRequestException("'Accept' header must be 'application/fhir+json'");
        }
```
This should also apply to the `headerPrefer` if statements below